gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -540,6 +539,7 @@ def run(argv=None):
             total_base_pairs, schema_file)
         logging.info('Integer range partitioned table %s was created.',
                      table_name)
+    suffixes.append(bigquery_util.SAMPLE_INFO_TABLE_SUFFIX)
```

Comment:
Somewhere around this line we need to create `sample_info` table:
```
bigquery_util.create_sample_info_table(known_args.output_table)
```

Reply:
I checked the failed tests and I found the culprit:
We treat `sample_info` table similar to other variant tables and that causes the failure (for example when we try to flatten it, it fails because it does not have a `call` column).

Here is the solution:
After AVRO load is done:
```
not_empty_variant_suffixes = load_avro.start_loading()
```
We need to remove `bigquery_util.SAMPLE_INFO_TABLE_SUFFIX` from both `suffixes` and `not_empty_variant_suffixes`.